.github/workflows/lint-core.yaml

```
@@ -6,12 +6,13 @@ on:
 
 permissions:
   contents: read
+            write
 
 jobs:
   lint-core:
     runs-on: ubuntu-latest
     strategy:
-      matrix:
+      matrix: true
         python-version: ["3.12"]
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)